crates/amaru-consensus/src/lib.rs

```diff
@@ -33,17 +33,17 @@ pub type RawHeader = Vec<u8>;
 pub enum ConsensusError {
     #[error("cannot build a chain selector without a tip")]
     MissingTip,
-    #[error("Failed to fetch block at {0:?}")]
+    #[error("Failed to fetch block at {}", .0.slot_or_default())]
     FetchBlockFailed(Point),
-    #[error("Failed to validate header at {0:?}: {1}")]
+    #[error("Failed to validate header at {}: {}", .0.slot_or_default(), .1)]
```

Comment:
There's a `Display` implementation for `Point`, would it not be simpler to use that all over the place?

Reply:
Error message would then be:

```
2025-07-03T09:36:22.636669Z ERROR amaru::stages::consensus::validate_header: failed to validate header Failed to validate header at Specific(61297725, [76, 250, 36, 173, 42, 219, 24, 43, 144, 81, 173, 40, 126, 156, 238, 34, 197, 234, 95, 176, 234, 51, 13, 193, 71, 189, 188, 196, 203, 189, 0, 110]): Insuficient leader stake.
```

Is this what you are looking for (cf your comment about the hash)?